base_classes/NXdetector.nxdl.xml

```diff
@@ -21,935 +21,982 @@
+        </doc>
+    </group>
+    <field name="solid_angle" type="NX_FLOAT" units="NX_SOLID_ANGLE">
+        <doc>
+             Solid angle subtended by the detector at the sample
+        </doc>
+        <dimensions rank="2">
+            <dim index="1" value="i"/>
+            <dim index="2" value="j"/>
+        </dimensions>
+    </field>
+    <field name="x_pixel_size" type="NX_FLOAT" units="NX_LENGTH">
+        <doc>
+             Size of each detector pixel. If it is scalar all pixels are the same size.
+        </doc>
+        <dimensions rank="2">
+            <dim index="1" value="i"/>
+            <dim index="2" value="j"/>
+        </dimensions>
+    </field>
+    <field name="y_pixel_size" type="NX_FLOAT" units="NX_LENGTH">
+        <doc>
+             Size of each detector pixel. If it is scalar all pixels are the same size
+        </doc>
+        <dimensions rank="2">
+            <dim index="1" value="i"/>
+            <dim index="2" value="j"/>
+        </dimensions>
+    </field>
+    <field name="dead_time" type="NX_FLOAT" units="NX_TIME">
+        <doc>
+             Detector dead time
+        </doc>
+        <dimensions rank="3">
+            <dim index="1" value="nP"/>
+            <dim index="2" value="i"/>
+            <dim index="3" value="j"/>
+        </dimensions>
+    </field>
+    <field name="gas_pressure" type="NX_FLOAT" units="NX_PRESSURE">
+        <doc>
+             Detector gas pressure
+        </doc>
+        <dimensions rank="2">
+            <dim index="1" value="i"/>
+            <dim index="2" value="j"/>
+        </dimensions>
+    </field>
+    <field name="detection_gas_path" type="NX_FLOAT" units="NX_LENGTH">
+        <doc>
+             maximum drift space dimension
+        </doc>
+    </field>
+    <field name="crate" type="NX_INT">
+        <doc>
+             Crate number of detector
+        </doc>
+        <dimensions rank="2">
+            <dim index="1" value="i"/>
+            <dim index="2" value="j"/>
+        </dimensions>
+        <attribute name="local_name">
+            <doc>
+                 Equivalent local term
+            </doc>
+        </attribute>
+    </field>
+    <field name="slot" type="NX_INT">
+        <doc>
+             Slot number of detector
+        </doc>
+        <dimensions rank="2">
+            <dim index="1" value="i"/>
+            <dim index="2" value="j"/>
+        </dimensions>
+        <attribute name="local_name">
+            <doc>
+                 Equivalent local term
+            </doc>
+        </attribute>
+    </field>
+    <field name="input" type="NX_INT">
+        <doc>
+             Input number of detector
+        </doc>
+        <dimensions rank="2">
+            <dim index="1" value="i"/>
+            <dim index="2" value="j"/>
+        </dimensions>
+        <attribute name="local_name">
+            <doc>
+                 Equivalent local term
+            </doc>
+        </attribute>
+    </field>
+    <field name="type">
```

Comment:
what is a difference between `detector:/type` and `detector:/detector_type` ? Shall not add a new concept if it is already there.